intera_core_msgs/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ add_message_files( DIRECTORY msg
         IOStatus.msg
         IONodeStatus.msg
         URDFConfiguration.msg
+        InteractionControlCommand.msg
```

Comment:
The InteractionControl messages can be sent directly to the Motor Controller plugin so we may want to keep them separate from the *_motion_msgs